.changeset/smooth-windows-relax.md

```diff
@@ -0,0 +1,6 @@
+---
+"@skip-go/client": patch
+"@skip-go/widget": patch
+---
+
+onTransactionSignRequested
```

Comment:
nit: not a big deal, but maybe change the changelog to say "added onTransactionSignRequested callback" or something